core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/OnCompletionForwarder.java

```diff
@@ -28,6 +28,11 @@ public void onRepeatModeChanged(@Player.RepeatMode int repeatMode) {
         // TODO: should we send?
     }
 
+    @Override
+    public void onShuffleModeEnabledChanged(boolean shuffleModeEnabled) {
```

Comment:
These listeners are less than ideal but we plan to work on tidying these up as per https://github.com/novoda/no-player/issues/95